user-registration-app/src/app/services/CustomErrorHandler/custom-error-handler.service.ts

```diff
@@ -89,7 +89,7 @@ export class CustomErrorHandlerService {
   handleV2HttpError(err: { error: IApiResponse<Error> }) {
     const error = { error: err.error, message: err.error.body.data.message };
     console.error(error);
-    // this.alerts.danger(error.message);
+    // this.toastrService.error(error.message);
```

Comment:
Is there a reason we're keeping this commented line and the console.log above it? Just wondering if these should be deleted since we're cleaning up everything anyway

Reply:
No idea, would there be a case where some error gets thrown and we'd want the participant to send a screenshot of it tho?

Reply:
Ideally, if there's an error we should be able to find it in the gcloud console. But I just kept it b/c idk how much we want the user to know about what went wrong and how easy we want to be able to debug it ourselves. I figured I'd leave it just in case